### PR TITLE
Add GHA for `ingest` "run_and_compare" CLI

### PR DESCRIPTION
**Pull request by sf-dcp**
### What
This is a GHA ro run the `run_and_compare` ingest migration command which: pulls data from source using library & ingest, does some pre-processing steps for the "ingest" version, uploads both versions to database, and runs a comparison between the two. 

### Why to create this GHA
While migration of recipe templates is a one-time adventure and we could do all of this locally, I had an issue with a particularly big dataset. My local machine just can't handle the size and crashes. Thus the reason for creating this GHA. 

### Things to keep in mind
It requires presence of both recipe templates (in ingest & library). Most likely, this action will be run from a feature branch.

This is an example of a job:

image

#### Successful GHA run [here](https://github.com/NYCPlanning/data-engineering/actions/runs/12284126901/job/34279185084#step:6:723). This is the dataset I couldn't run locally by the way.

### CODE REVIEW

File: .github/workflows/ingest_library_compare.yml [hunk 1]
```diff
@@ -11,19 +11,33 @@ on:
       schema_name:
```
Comment (alexrichey) [inline .github/workflows/ingest_library_compare.yml:11]: oh, sorry, forgot to ask about this, but one line up: `default: dcp_mappluto` do we want that?

**Review by alexrichey — APPROVED** [review]
One question, but looks good. Is there a link to a successful run? Curious what the output would look like for a diff like this.

**Comment by sf-dcp** [comment]
> One question, but looks good. Is there a link to a successful run? Curious what the output would look like for a diff like this.

Yes! Please see the link in my last sentence of the issue description.

File: .github/workflows/ingest_library_compare.yml (hunk 1 above)
Comment (sf-dcp) [reply .github/workflows/ingest_library_compare.yml:11]: Eh, not really. Will remove

**Pull request merged**